<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySimIIGui::Boundariess</class>
 <widget class="QWidget" name="MySimIIGui::Boundariess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Boundary settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QTabWidget" name="TabWid_1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="TabContent1">
        <attribute name="title">
         <string>Flowing</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_Tab1">
         <item row="3" column="0">
          <widget class="QLabel" name="label_Pressure">
           <property name="text">
            <string>Pressure(N/m^2)：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_VelY">
           <property name="text">
            <string>Vel.Y(m/s)：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_VelY"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_NTCoord">
           <property name="text">
            <string>NTCoord：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_VelX"/>
         </item>
         <item row="6" column="1">
          <widget class="QCheckBox" name="Wall">
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_FreeSurf">
           <property name="text">
            <string>FreeSurf：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_VelZ">
           <property name="text">
            <string>Vel.Z(m/s)：</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_Wall">
           <property name="text">
            <string>Wall：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_VelX">
           <property name="text">
            <string>Vel.X(m/s)：</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QCheckBox" name="FreeMoving">
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="NTCoord">
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QCheckBox" name="FreeSurf">
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_Pressure"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_SlipCoeff">
           <property name="text">
            <string>SlipCoeff：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_VelZ"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_FreeMoving">
           <property name="text">
            <string>FreeMoving：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_SlipCoeff">
           <property name="text">
            <string>0.00002</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TabContent2">
        <attribute name="title">
         <string>Heating</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_Tab2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_Temperature">
           <property name="text">
            <string>Temp.(^oC)：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_Temperature"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_HeatFlx.">
           <property name="text">
            <string>HeatFlx(Watt/m^2/s)：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_HeatFlux"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_Blank1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TabContent3">
        <attribute name="title">
         <string>EM-Waving</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_Tab3">
         <item row="5" column="0">
          <widget class="QLabel" name="Labal_ForcingType">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Forcing type:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QComboBox" name="ComB_EM">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Empty</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PEC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rad.</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WavePort</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LumpPort</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Inc.Wave</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_IncWaveX">
           <property name="text">
            <string>IncWaveX：</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEdit_IncWaveX">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_IncWaveY">
           <property name="text">
            <string>IncWaveY：</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="lineEdit_IncWaveY">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_IncWaveZ">
           <property name="text">
            <string>IncWaveZ：</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="lineEdit_IncWaveZ">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_IncPropX">
           <property name="text">
            <string>IncPropX：</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="lineEdit_IncPropX">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_IncPropY">
           <property name="text">
            <string>IncPropY：</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLineEdit" name="lineEdit_IncPropY">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_IncPropZ">
           <property name="text">
            <string>IncPropZ：</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLineEdit" name="lineEdit_IncPropZ">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TabContent4">
        <attribute name="title">
         <string>Self-Defining</string>
        </attribute>
        <widget class="QLabel" name="Labal_GeometryInfo">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>127</width>
           <height>13</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Parameters and functions:</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit_SelfDef1">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>28</y>
           <width>251</width>
           <height>71</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="text" stdset="0">
          <string>## Geometry Info ##</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_Blank2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>130</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="Labal_SelfDefinition2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>110</y>
           <width>69</width>
           <height>16</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Self definition:</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit_SelfDef2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>251</width>
           <height>71</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="text" stdset="0">
          <string>## including as Elmer MATC function.</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
